crates/katana/primitives/src/utils/transaction.rs

```diff
@@ -255,7 +256,6 @@ pub fn compute_l1_handler_tx_hash(
         contract_address,
         entry_point_selector,
         compute_hash_on_elements(calldata),
-        FieldElement::ZERO, // No fee on L2 for L1 handler tx
```

Comment:
@kariy if you could double confirm this modification. I think it was a typo since the first implem, and never reviewed it since the starknet doc was updated.

Thanks mate. 🙏

Reply:
i believe including the fee there is already correct, according to these clients:

deoxys:

https://github.com/KasarLabs/deoxys/blob/82c49acdaa1167bc8dc67a3f6ad3d6856c6c7e89/crates/primitives/transactions/src/compute_hash.rs#L142-L151

pathfinder:

https://github.com/eqlabs/pathfinder/blob/677fd40fbae7b5b659bf169e56f055c59cbb3f52/crates/common/src/transaction.rs#L556 

https://github.com/eqlabs/pathfinder/blob/677fd40fbae7b5b659bf169e56f055c59cbb3f52/crates/common/src/transaction.rs#L768-L770

juno:

https://github.com/NethermindEth/juno/blob/d9e64106a3a6d81d217d3c8baf28749f4f0bdd71/core/transaction.go#L561-L569

Reply:
I've also based my comment on Madara, but actually interesting that all the others are not including it.
Will ask for a clarification, thanks for pinging this up mate.